pkg/logging/scrubbingLogWriter.go

```diff
@@ -72,18 +83,16 @@ func addMandatoryMasking(dict ScrubbingDict) ScrubbingDict {
 }
 
 func (w *scrubbingLevelWriter) Write(p []byte) (int, error) {
-	_, err := w.writer.Write(scrub(p, w.scrubDict, w.regexCache))
+	_, err := w.writer.Write(scrub(p, w.regexCache))
 	return len(p), err // we return the original length, since we don't know the length of the redacted string
 }
 
-func scrub(p []byte, scrubDict ScrubbingDict, regexCache map[string]*regexp.Regexp) []byte {
+func scrub(p []byte, regexCache map[string]*regexp.Regexp) []byte {
 	s := string(p)
-	for term := range scrubDict {
+	for term, re := range regexCache {
 		if len(term) > 0 {
```

Comment:
When would `term` be an empty string? Could we prevent those from getting into the cache in the first place?

Reply:
Totally.